src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -0,0 +1,287 @@
+process.env.DISABLE_INIT_RANDOM_SEED = "true";
+
+const _ = require('lodash');
+const path = require('path');
+const P = require('../../../util/promise');
+const fs_utils = require('../../../util/fs_utils');
+const os_util = require('../../../util/os_utils');
+const nb_native = require('../../../util/nb_native');
+
+const MAC_PLATFORM = 'darwin';
+let tmp_fs_path = '/tmp/test_bucketspace_fs';
+if (process.platform === MAC_PLATFORM) {
+    tmp_fs_path = '/private/' + tmp_fs_path;
+}
+
+const DEFAULT_FS_CONFIG = {
+    uid: process.getuid(),
+    gid: process.getgid(),
+    backend: '',
+    warn_threshold_ms: 100,
+};
+
+const nc_nsfs_manage_actions = {
+    ADD: 'add',
+    UPDATE: 'update',
+    DELETE: 'delete'
+};
+
+describe('manage nsfs cli account flow', () => {
+    const accounts_schema_dir = 'accounts';
+    const access_keys_schema_dir = 'access_keys';
+
+    describe('cli create account', () => {
+        const config_root = path.join(tmp_fs_path, 'config_root_manage_nsfs');
+        const root_path = path.join(tmp_fs_path, 'root_path_manage_nsfs/');
+        const defaults = {
+            type: 'account',
+            name: 'account1',
+            email: 'account1@noobaa.io',
+            new_buckets_path: `${root_path}new_buckets_path_user1/`,
+            uid: 999,
+            gid: 999,
+            access_key: 'abc',
+            secret_key: '123',
+        };
+
+        beforeEach(async () => {
+            await P.all(_.map([accounts_schema_dir, access_keys_schema_dir], async dir =>
+                fs_utils.create_fresh_path(`${config_root}/${dir}`)));
+            await fs_utils.create_fresh_path(root_path);
+        });
+
+        afterEach(async () => {
+            await fs_utils.folder_delete(`${config_root}`);
+            await fs_utils.folder_delete(`${root_path}`);
+        });
+
+        it('cli create account without access_keys', async () => {
+            const action = nc_nsfs_manage_actions.ADD;
+            const { type, name, email, new_buckets_path, uid, gid } = defaults;
+            const account_options = { config_root, name, email, new_buckets_path, uid, gid };
+            await fs_utils.create_fresh_path(new_buckets_path);
+            await fs_utils.file_must_exist(new_buckets_path);
+            await exec_manage_cli(type, action, account_options);
+            const account = await read_config_file(config_root, accounts_schema_dir, name);
+            assert_account(account, account_options, false);
+            const access_key = account.access_keys[0].access_key;
+            const secret_key = account.access_keys[0].secret_key;
+            expect(access_key).toBeDefined();
+            expect(secret_key).toBeDefined();
+            const account_symlink = await read_config_file(config_root, access_keys_schema_dir, access_key, true);
+            assert_account(account_symlink, account_options);
+        });
+
+        it('cli create account with access_key and without secret_key', async () => {
```

Comment:
I'm not sure this a valid behavior, are you sure we want to generate only access key / secret_key?

Reply:
Well, as we are set now, it is possible regardless of the regenerate code. 
There is no limitation to update only Paris and we can do an update with only one accessKey or secretKey
This code is just following it.
We can create verification that will handle it (again regardless of this PR).
If we want to we should open an issue and do a PR accordingly.

For this PR scope, I think we should keep it (and decide later if we want to handle it).
One more thought is, IMHO, we should not allow the user to give those values at all, and we should actually always generate or regenerate both keys (and remove the access_key/secret_key/new_access_kek completely).